src/main/resources/default/taglib/t/associatedProcesses.html.pasta

```diff
@@ -0,0 +1,41 @@
+<i:arg type="sirius.db.mixing.BaseEntity" name="entity"
+       description="The entity for which a list of associated process links shall be generated."/>
+
+<t:navbox labelKey="Process.associatedProcesses">
+    <i:local name="processes" value="@part(sirius.biz.process.Processes.class).fetchAssociatedProcesses(entity)"/>
+    <i:for var="process"
```

Comment:
Gibt immer mindestens einen Prozess?

Reply:
Nein, nur laufenden und für den User sichtbare Prozesse. Wieso? Ist halt n loop

Reply:
ich glaub das FW fängt leere nabox tags ab... sonst könnte man n if drumrum pappen, haben die liste ja schon in ner variable...

Reply:
Ja das navbox Template macht einen Check ob überhaupt ein Body/Inhalt generiert wurde